modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1241,6 +1241,113 @@ Suggestions for improvement::
 Use `DATABASE *` without the parameter to revoke the privilege on all databases.
 ====
 
+[#_neo_clientnotification_security_authprovidernotdefined]
+=== AuthProviderNotDefined
+
+.Notification details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Security.AuthProviderNotDefined
+|Title
+a|The auth provider is not defined.
+|Description
+a|The auth provider `<provider>` is not defined in the configuration.
+Verify that the spelling is correct or define `<provider>` in the configuration.
+|Severity
+m|INFORMATION
+|Category
+m|SECURITY
+|===
+
+.Create a user with an auth provider that isn't defined in config
+====
+Command::
++
+[source, cypher]
+----
+CREATE USER foo SET AUTH 'unknownProvider' { SET ID 'idString' }
+----
+
+Description of the returned code::
+The auth provider `unknownProvider` is not defined in the configuration.
+Verify that the spelling is correct or define `unknownProvider` in the configuration.
+
+Suggestions for improvement::
+Make sure that the given provider is correct, or replace it if not.
+If it is correct, make sure to add it as a known auth provider in one or both of `dbms.security.authentication_providers` and `dbms.security.authorization_providers`.
+====
+
+.Alter a user to add an auth provider that isn't defined in config
+====
+Command::
++
+[source, cypher]
+----
+ALTER USER foo SET AUTH 'unknownProvider' { SET ID 'idString' }
+----
+
+Description of the returned code::
+The auth provider `unknownProvider` is not defined in the configuration.
+Verify that the spelling is correct or define `unknownProvider` in the configuration.
+
+Suggestions for improvement::
+Make sure that the given provider is correct, or replace it if not.
+If it is correct, make sure to add it as a known auth provider in one or both of `dbms.security.authentication_providers` and `dbms.security.authorization_providers`.
+====
+
+[#_neo_clientnotification_security_externalauthnotenabled]
+=== ExternalAuthNotEnabled
+
+.Notification details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Security.ExternalAuthNotEnabled
```

Comment:
This notification was an error in CIP-145 but was decided to be just a warning during implementation (to not lock out current external only users from being able to switch by first adding the new external only users to system before disabling their ways to log in)